proto/ate.proto

```diff
@@ -643,6 +643,13 @@ message Ipv4Header {
   uint32 dscp = 4;
   uint32 ecn = 5;
   uint32 checksum = 7;
+  uint32 header_length = 8;
+  UIntRange protocol = 9;
+  uint32 total_length = 10;
+  bool identification = 11;
```

Comment:
Since an identification field is 16 bits, what was the rationale for making this a boolean as opposed to an integer?

Reply:
updated it with 16 bits.